src/main/scala/de/m7w3/signal/AccountHelper.scala

```diff
@@ -13,11 +13,10 @@ import org.whispersystems.libsignal.util.{KeyHelper, Medium}
 import org.whispersystems.signalservice.api.SignalServiceAccountManager
 import org.whispersystems.signalservice.internal.util.Base64
 
-case class AccountHelper(userId: String){
+case class AccountHelper(userId: String, password: String = Util.getSecret(20)){
```

Comment:
shouldnt this password be somewhat static and not change between each instantiation? So, chose a password once at registration and keep it? using a default argument value here seems way too implicit.